src/modules/navigation/sections/Navigation.vue

```diff
@@ -201,5 +201,14 @@ export default {
 		margin-top: 3vh;
 	}
 }
+</style>
 
+<style>
+.archived-items .app-navigation-entry__children {
+	& .app-navigation-entry-wrapper .app-navigation-entry__children {
+		& .app-navigation-entry {
+			padding-left: calc(16px * 2);
```

Comment:
Any specific reason to not just use 32px?

Reply:
I figured it would be clearer to use that since I did not have access to the $icon-size variable which is defined in the Vue library and set at 16px, to make it clearer that that's what I was referencing. But yeah this can be changed

Reply:
All right, would have been fine either way, in that case a code comment might have made sense explaining the calculation, but fine to keep the 32px as well 👍